test/integration/advertise_change_test.lua

```diff
@@ -0,0 +1,86 @@
+local fio = require('fio')
+local fun = require('fun')
+local t = require('luatest')
+local g = t.group()
+
+local helpers = require('test.helper')
+
+local function move(srv, uri)
+    srv.env['TARANTOOL_ADVERTISE_URI'] = uri
+    srv.net_box_uri = uri
+end
+
+g.before_all(function()
+    g.cluster = helpers.Cluster:new({
+        datadir = fio.tempdir(),
+        use_vshard = false,
```

Comment:
Why not?

Reply:
For faster tests. It's not related to this case